Formula/postgis.rb

```diff
@@ -3,6 +3,8 @@ class Postgis < Formula
   homepage "https://postgis.net/"
   url "https://download.osgeo.org/postgis/source/postgis-3.0.2.tar.gz"
   sha256 "a3a1641dfd73c83924088a185bdb8b35567b3d1dc8d0887f9e4b492e228ab2ca"
+  license "GPL-2.0-or-later"
+  revision 1
```

Comment:
can you amend your commit message as `postgis: <why need a revision bump>`?

Reply:
Did not build against postgresql 13.0, so bumped rev on request.
Added license conforming to what is mentioned on the postgis homepage.

Reply:
Sounds good  👍